core/src/main/java/com/scalar/db/storage/cassandra/SelectStatementHandler.java

```diff
@@ -55,11 +56,13 @@ public SelectStatementHandler(Session session) {
   @Override
   @Nonnull
   protected PreparedStatement prepare(Operation operation) {
-    checkArgument(operation, Get.class, Scan.class);
+    checkArgument(operation, Get.class, Scan.class, ScanAll.class);
```

Comment:
If we add `ScanAll.class` here, I think we should add the same to the following place:
https://github.com/scalar-labs/scalardb/pull/570/files#diff-5feb096f3b977bcff460e21a574fbb6b8a0f8f84a0540f84f63eda0be2a14e5cR76

Or we don't need it for both places?

Reply:
Indeed, it's not necessary in both cases. I will remove this. 
Thank you.

Reply:
Sorry, I spoke too fast. 😅 
After checking more in detail, Since I added specific `prepare()` method for ScanAll 
 ` private Select prepare(ScanAll scanAll)`. I think it's more cautious to keep the `checkArgument(operation, Get.class, Scan.class, ScanAll.class);` as is.